bigbluebutton-html5/imports/ui/components/modal/simple/styles.scss

```diff
@@ -22,6 +22,11 @@
   flex-shrink: 0;
 }
 
+.headerNoBorder {
```

Comment:
You are also duplicatin styles.

You should compose them by in the `hideNoBorder` only overwriting the properties you want to change.